core/templates/ingest.yml

```diff
@@ -102,6 +102,11 @@ keys:
         type: str
         filter_by: '^[a-zA-Z](\w+)?[a-zA-Z0-9]$'
 
+    # used for nuke retime script
+    ingestRetimeName:
+        type: str
+        filter_by: '^(\w+_rt)$'
```

Comment:
It should atleast start with a character `^([a-zA-Z]\w+_rt)$`

Reply:
i will update the regex